https://bugs.gentoo.org/927137
https://github.com/OpenPrinting/cups-filters/issues/578
https://github.com/OpenPrinting/libcupsfilters/commit/107091186dce1c0cb2f042f8b880f571089acaf1

From 107091186dce1c0cb2f042f8b880f571089acaf1 Mon Sep 17 00:00:00 2001
From: zdohnal <zdohnal@redhat.com>
Date: Wed, 29 Nov 2023 13:30:48 +0100
Subject: [PATCH] raster.c: Fix build after last fix (#40)

Add header file `stdbool`, since we started to use `bool` in
`raster_base_header()`.
---
 cupsfilters/raster.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cupsfilters/raster.c b/cupsfilters/raster.c
index 5e10f3bb..e9a8ad1f 100644
--- a/cupsfilters/raster.c
+++ b/cupsfilters/raster.c
@@ -29,6 +29,7 @@
 #include <cupsfilters/ipp.h>
 #include <cupsfilters/libcups2-private.h>
 #include <cups/pwg.h>
+#include <stdbool.h>
 
 //
 // Local functions

